Update Module.php to fix assets register

niciz opened the pull request:
Register assets using the yii\web\View functions.
In my application I have override the standard yii\web\View in order to add CSP nonce to each scripts loaded with View functions. Unfortunately yii-debug module doesn't use those functions to load assets. This is the fix.

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | none

samdark commented:
@niciz I haven't seen extra commits. Now it looks alright. Would you please add a line for CHANGELOG? Thanks.

samdark commented:
Also please merge `master` into your branch.

niciz commented:
Hi @samdark sorry for late reply.
Yes I fixed it and currently I'm using it in production without problems.
Merged and added changes to CHANGELOG.md

Let me know if you need something else.

samdark commented:
Seems tests are broken because of the changes. Would you please check these?
